docs/release-notes/release_notes_index.md

```diff
@@ -1,6 +1,6 @@
 # Percona Everest release notes index
 
-- [Percona Everest 1.0.1 (2024-07-08)](Percona-Everest-1.0.1-(2024-07-08).md)
```

Comment:
This shouldn't have been removed, right? We still had 1.0.1, now we're adding 1.1.0...

Reply:
Yes. Removed accidentally...